rk3-4

UmnAlAl opened the pull request:
Все что готово к рк в четверг.

UmnAlAl committed:
frontenders requierments

-bugs in servlets fixed

UmnAlAl committed:
game basics

UmnAlAl committed:
night commit pre rk

game extends...

UmnAlAl committed:
game improvements

UmnAlAl committed:
small game improvements

UmnAlAl committed:
properties for server & db

UmnAlAl committed:
rk

UmnAlAl committed:
del output

UmnAlAl committed:
delete output

UmnAlAl committed:
game basics

UmnAlAl committed:
tmp game for frontend rk v1

Vertaler committed:
Исправлена ошибка в файле game/game_objects/blocks

Vertaler committed:
Поправлено получение данных из сокета.

Vertaler committed:
Исправлена toJSON у gamestate

Vertaler committed:
Отправка стейта с каждым действием.

UmnAlAl committed:
[for rk frontend] works!!!

UmnAlAl committed:
123

UmnAlAl committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	public_html/js/models/gamestate.js

UmnAlAl committed:
what am i doing...

Vertaler committed:
Добавлен toJSON объектам канваса

UmnAlAl committed:
lalala

UmnAlAl committed:
lalala

UmnAlAl committed:
normal game implementation

UmnAlAl committed:
improvements

UmnAlAl committed:
disconnect

UmnAlAl committed:
player list

Vertaler committed:
fix

UmnAlAl committed:
pre rk

UmnAlAl committed:
rk

KSolovyev commented:
папку output удалить из репозитория и в .gitignore.

UmnAlAl commented:
Не понимаю, откуда удалить.
В репозитории папки output нет.
https://github.com/LoremIpsum2016/2016-02-Lorem-Ipsum
В .gitignore есть
https://github.com/LoremIpsum2016/2016-02-Lorem-Ipsum/blob/master/.gitignore

KSolovyev commented:
Ок, они оказываются удалены

Vertaler committed:
Удалены файлы фронтенда

oxeg commented:
game/gamemanagement/websocket/GameWebSocket.java - горит огромное количество инспекций.
И система сообщений сделана непонятно зачем. Сейчас у вас единственный сервис, который принимает сообщения извне, никак на них не отвечая. При таком раскладе система сообщений - излишнее усложнение. Проще тогда уж обращаться к этому сервису напрямую.

UmnAlAl committed:
Interaction of threads via MessageSystem

Message input (does smth and sends mess) and output processors interact
via MessageSystem. Input ms prc does smth and (possibly) sends messages
to output msg prc. Output msg prc sends responses to the user.

UmnAlAl commented:
Добавлен пример двух взаимодействующих сервисов.
Исправлено большинство инспекций.

UmnAlAl committed:
issues

UmnAlAl commented:
Исправление issues.
https://github.com/java-park-mail-ru/2016-02-Lorem-Ipsum/issues

d-frolov commented:
компиляция сломана
